README.org

```diff
@@ -1,40 +1,55 @@
 #+CONTEXT_HEADER_EXTRA: \setuplayout[backspace=36pt]
-#+TITLE: Readme
+#+TITLE: ox-context
 
-* About
+This is a [[https://wiki.contextgarden.net][ConTeXt]] [[https://orgmode.org/manual/Exporting.html][exporter]] for [[https://orgmode.org/][Org Mode]].
 
-This is a [[https://wiki.contextgarden.net/Main_Page][ConTeXt]] exporter for [[https://orgmode.org/][Org Mode]].
+* Installation
+Clone the repo to the desired location, then add that directory to your load-path:
+
+#+BEGIN_SRC sh
+git clone https://github.com/Jason-S-Ross/ox-context
+#+END_SRC
+
+Then, in your customization file, add
+#+BEGIN_SRC elisp
+(add-to-list 'load-path "~/path-to-repo/ox-context/")
+#+END_SRC
+
+* Examples
+See [[https://github.com/Jason-S-Ross/ox-context-examples][the example repo]] for examples.
 
 * Document Customization
 
-Each element type in the Org document is associated with a custom ConTeXt
-command whose name and definition is set by a customization variable. When you
-export a document, the command definition associated with each type of org
-element is exported into the ConTeXt document preamble. The element itself is
-formatted by the associated command. This allows documents to be customized by
-either changing the customization variable associated with each element type, or
-by inserting new command definitions into the document preamble after the
-original definitions in order to override them. There are several ways to insert
-command definitions into the preamble:
+Each element type in the Org document is associated with a custom ConTeXt command
+whose name and definition is set by a customization variable.
+When you export a document,
+the command definition associated with each type of org element
+is exported into the ConTeXt document preamble.
+The element itself is formatted by the associated command.
+This allows documents to be customized
+by either changing the customization variable associated with each element type,
+or by inserting new command definitions into the document preamble
+after the original definitions in order to override them. 
+here are several ways to insert command definitions into the preamble:
```

Comment:
"here are several ways" should be "Here are several ways" or "There are several ways"